backend/exchanges/btcdirect.go

```diff
@@ -80,14 +80,21 @@ func BtcDirectDeals() *ExchangeDealsList {
 		ExchangeName: BTCDirectName,
 		Deals: []*ExchangeDeal{
 			{
-				Fee:     3,
+				Fee:     3, // 3%
 				Payment: CardPayment,
-				IsFast:  true,
```

Comment:
I think we should keep the isFast here like for the other card payments (e.g. moonpay)

Reply:
ack. fixedup and force pushed.

Reply:
fixed, @Beerosagos PTAL should be all good now to merge this